### PR TITLE
Sentry 7.4.x

### PR DESCRIPTION
**Pull request by dzen**

### CODE REVIEW

File: autoguard/config_helpers.py [hunk 1]
```diff
@@ -62,42 +68,34 @@ def mail_from_url(url):
...
+            'django_caches': {'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}},
+        },
+        'memcached': {
+            'sentry': 'sentry.cache.django.DjangoCache',
+            'buffer': 'sentry.buffer.base.Buffer',  # TODO: check alternative backends
+            'tsdb': 'sentry.tsdb.dummy.DummyTSDB',
+            'quota': 'sentry.quotas.Quota',
+            'rate_limits': 'sentry.ratelimits.base.RateLimiter',
+
+            'redis_options': {},
```
Comment (vperron) [inline autoguard/config_helpers.py]: dans memcached ?

File: autoguard/config_helpers.py [hunk 2]
```diff
@@ -53,7 +58,8 @@ def mail_from_url(url):
         'host': parsed.hostname,
         'port': parsed.port,
         'use_tls': False,
-        'sender': "no-reply@{}".format(parsed.hostname if parsed.hostname else 'localhost')
+        'sender': "no-reply@{}".format(parsed.hostname if parsed.hostname else 'localhost'),
```
Comment (vperron) [inline autoguard/config_helpers.py]: tant qu'a faire, mettre ça dans get_hostname ?

File: autoguard/ldap.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# SSO using LDAP to get access rights
```
Comment (vperron) [inline autoguard/ldap.py]: utf-8

File: autoguard/ldap.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    @staticmethod
+    def _add_to_all_organizations(user):
+        if user is None:
+            return
+
+        organizations_to_add = (
+            Organization.objects
+            .order_by('pk')
+            .exclude(members=user)
```
Comment (vperron) [inline autoguard/ldap.py]: members__user=user non ?

File: autoguard/config_helpers.py (hunk 1 above)
Comment (fbochu) [inline autoguard/config_helpers.py]: Yes, it is the memcached config as you can see on line 95.

File: autoguard/config_helpers.py (hunk 2 above)
Comment (fbochu) [inline autoguard/config_helpers.py]: It's a good idea !

File: autoguard/ldap.py (hunk 4 above)
Comment (fbochu) [inline autoguard/ldap.py]: No, just `members`, `members_user` does not work.

Comment (vperron) [inline autoguard/ldap.py]: my bad, I thought organization.memebers were OrganizationMember objects, but they are users.

**Pull request merged**